### PR TITLE
WixStorageStrategy: Injectable `HttpClient` instance

### PR DESCRIPTION
**Pull request by kobiburnley**
Allow passing a ready `httpClient` to `WixStorageStrategy`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,14 +8,15 @@
   "main": "dist/src/index.js",
   "module": "dist/es/src/index.js",
   "sideEffects": false,
-  "types": "dist/src/typings/index.d.ts",
+  "types": "dist/src/index.d.ts",
```
Comment (kobiburnley) [inline package.json:11]: Project is now written with TS, can provide typescript generated by `tsc`

File: src/index.ts [hunk 2]
```diff
@@ -11,8 +11,9 @@ export { FrameStorageStrategy } from './strategies/frame-storage';
 export { InMemoryStorageStrategy } from './strategies/in-memory-storage';
 export { LocalStorageStrategy } from './strategies/local-storage';
 export type { LocalStorageStrategyOptions } from './strategies/local-storage';
-export { WixStorageStrategy } from './strategies/wix-storage';
+export { WixStorageAxiosStrategy as WixStorageStrategy } from './strategies/wix-storage-axios';
```
Comment (kobiburnley) [inline src/index.ts:14]: Not breaking API, exporting `WixStorageAxiosStrategy` as `WixStorageStrategy`

**Review by roymiloh — COMMENTED** [review]
Nice! please add a test.

**Comment by kobiburnley** [comment]
@roymiloh https://github.com/wix-private/fe-essentials/pull/1326

**Pull request closed without merging**